pkgs/desktops/expidus/file-manager/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, flutter, fetchFromGitHub }:
+with lib;
+let
+  version = "0.1.2";
+in flutter.buildFlutterApplication {
+  pname = "expidus-file-manager";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "ExpidusOS";
+    repo = "file-manager";
+    rev = version;
+    sha256 = "sha256-aAPmwzNPgu08Ov9NyRW5bcj3jQzG9rpWwrABRyK2Weg=";
+  };
+
+  depsListFile = ./deps.json;
+  vendorHash = "sha256-mPGrpMUguM9XAYWH8lBQuytxZ3J0gS2XOMPkKyFMLbc=";
+
+  postInstall = ''
+    rm $out/bin/file_manager
+    ln -s $out/app/file_manager $out/bin/expidus-file-manager
```

Comment:
```suggestion
    ln -fs $out/app/file_manager $out/bin/expidus-file-manager
```

Reply:
Actually, that keeps 2 executables which that shouldn't be done.
```
$ ls result/bin
expidus-file-manager  file_manager
```